### ISSUE TITLE
Skinning bug: Worn Dragonscale does not drop + sometimes skinning loot produces nothing

### ISSUE THREAD

**Issue by matanre** [dalaranwow/dalaran-wow#5304]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: When skinning, on some occasions the loot from the skinning is nothing (instead of heavy leather, for example).
Moreover, after a few dozen if not hundreds of skinning of Dragonkins in Dustwallow Marsh, not a single [Worn Dragonscale] had dropped.
ID - Item=8165
*I am assuming that they should have dropped when nothing was looted, but this is only an assumption.

**Expected behaviour**: Skinning should always produce at least 1 type of leather / hide / scale etc.
AND Worn Dragonscales should have some sort of drop rate in Dustwallow Marsh,

**Steps to reproduce the problem**:

1. Kill some dragonkin / dragon whelps in Dustwallow Marsh.
2. Skin them.
3. Sometimes nothing will drop.

*I would like to add that the "nothing drops" part had happened to me in other zones as well, but was much more common when looking for dragon scales,

*If any screenshot is necessary please tell me and I will try to get some, although it is a little problematic since it is a screen shot of "nothing skins" which doesn't happen all the time.

**Comment by AlexCSCP** [comment]
The drop rate of scales in Dustwallow Marsh is pretty low, like 1-3%, so you may kill hundreds and never see one.
They have a way higher drop rate in Burning Steppes and Sunken Temple (almost 100% from bosses).

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/1120
